test/cql-pytest/conftest.py

```diff
@@ -19,6 +20,13 @@
 
 from util import unique_name, new_test_table, cql_session
 
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(format='%(asctime)s.%(msecs)03d %(levelname)s> %(message)s', level=logging.DEBUG)
```

Comment:
Do we really need `basicConfig` here? I though that `pytest.ini` is the main tool to configure logs in `pytest`.

Reply:
I personally prefer this approach (the code in conftest.py). What's the point of moving a small part of the logic into a separate file pytest.ini? It's not like we can move anything else there, so what's the benefit of moving this specific command? 
Note that currently test/cql-pytest/pytest.ini is currently empty.

Reply:
By the way, I'm really curious what sort of benefit do you guys see in using "logging" in pytest over just using "print" everywhere. pytest already hides all the output unless the test fails, which is what you almost always want to do anyway - the concept of "log levels" seems to me a mismatch to testing.